src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,40 @@
+.catalog {
+  padding: 50px 40px;
+
+  display: grid;
+  grid-template-columns: repeat($column-count, $column-width);
```

Comment:
Ensure that the `$column-width` variable is defined in your SCSS files. If it's not, you need to define it to avoid compilation errors.